### PR TITLE
Teach pulp_export() to cast repositories to their Detail for plugin use

### PR DESCRIPTION
**Pull request by ggainey**

### CODE REVIEW

File: CHANGES/6466.bugfix [hunk 1]
```diff
@@ -0,0 +1 @@
+Taught pulp_export() to cast() repositories so plugins start with the right Detail type
```
Comment (daviddavis) [inline CHANGES/6466.bugfix:1]: I think we should probably just drop this changelog since 6466 hasn't been released yet anyway.

Comment (daviddavis) [reply CHANGES/6466.bugfix:1]: Nevermind. I see that 6466 is a new issue.

Comment (ggainey) [reply CHANGES/6466.bugfix:1]: 6466 isn't 6135 - this is a new issue. And pulpbot complains at me if "fixes #1234" exists but there is not CHANGES for 1234

Comment (ggainey) [reply CHANGES/6466.bugfix:1]: overlapping comments ftw!

**Comment by ggainey** [comment]
Closing this PR - this turns out to be Not The Problem

**Pull request closed without merging**